client/lessVariables.js

```diff
@@ -1,8 +1,14 @@
 module.exports = {
   '@primary-color': '#E8505B',
+  '@white': '#FFFFFF',
+  '@light-grey': '#F0F2F9',
```

Comment:
I searched for **@light-grey** and **@grey** in [default.less](https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less) But I did not find it